packages/tools-iottwinmaker/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@iot-app-kit/ts-config/base.json",
   "include": ["src"],
+  "exclude": ["src/**/*.spec.ts", "src/commands/test-constants.ts", "src/commands/test-utils.ts"],
```

Comment:
blocker: need to run typescript on tests

Reply:
I added this line so that the spec files were not included in the packaged executable, will discuss with you offline how to do this better

Reply:
check out the pattern used in other packages with a `tsconfig.build.json` file